eventing/samples/cronjob-source/README.md

```diff
@@ -0,0 +1,58 @@
+# Cron Job Source example
+
+Cron Job Source example shows how to configure Cron Job as event source for functions.
+
+## Deployment Steps
+
+### Prerequisites
+
+1. Setup [Knative Serving](https://github.com/knative/docs/tree/master/serving).
+1. Setup
+   [Knative Eventing](https://github.com/knative/docs/tree/master/eventing).
+
+### Channel
+
+1. Create a `Channel`. You can use your own `Channel` or use the provided sample, which creates a channel called `cronjob-test`. If you use your own `Channel` with a different name, then you will need to alter other commands later.
+
+```shell
+kubectl apply -f channel.yaml
+```
+
+### Create Cron Job Event Source
+
+1. In order to receive events, you need to create a concrete Event Source for a specific namespace. If you want to consume events from a differenet namespace or using a different `Service Account`, you need to modify the yaml accordingly.
+
+```shell
+kubectl apply -f cronjob-source.yaml
+```
+
+### Subscriber
+
+In order to check the `CronJobSource` is fully working, we will create a simple Knative Service that dumps incoming messages to its log and create a `Subscription` from the `Channel` to that Knative Service.
```

Comment:
You should be able to directly create the Service and then point the Source at it (and, if you follow the GitHub case, you might actually deploy the Service first).